aprsd_webchat_extension/web/chat/static/css/tour.css

```
@@ -0,0 +1,313 @@
+    width: 100% !important;
+    box-sizing: border-box !important;
+}
+
+.tour-progress {
+    font-size: 0.8125rem;
+    color: var(--text-secondary);
+    font-weight: 500;
+    margin-left: auto !important;
+    margin-right: 0 !important;
+    padding-left: 1rem !important;
+    margin-left: 1.5rem !important;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

margin-left is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)